src/components/tasks/TaskStatusDropdown.tsx

```diff
@@ -104,26 +116,55 @@ export default function TaskDropDown({
         onChange(payload);
         setMessage('');
     };
+
+    type TaskStatusItem = [string, string];
```

Comment:
- this is not required as the type for taskStatus will get inherited from our definition in `TaskStatusSelectProps`